### PR TITLE
Planning chapters (up to learning) typos and fixes

### PR DESCRIPTION
**Pull request by ynahum**

### CODE REVIEW

File: chapter2-ddp.tex [hunk 1]
```diff
@@ -305,7 +305,8 @@ \subsection{Reduction between control policies classes}
 respect to the randomness of the policy $\policy$. Now we can
 rewrite the expected cost to go as,
 \[
-\E[\Cost^\policy(\state_0)]=\sum_{\ttime=1}^{\tHorizon-1}
+\E[\Cost^\policy(\state_0)]=\sum_{\action\in \Actions_0}\cost_0(\state_0,\action)\pi
```
Comment (avivt) [inline chapter2-ddp.tex:308]: good catch. But this will be cleaner if we simply start the sum from t=0.

File: chapter2-ddp.tex [hunk 2]
```diff
@@ -418,9 +419,9 @@ \subsection{Reduction between control policies classes}
 \E^\policy_{\state_\ttime}[\Cost_{\ttime}(\state_\ttime)]
 %=\E^\policy[\Cost(\state_\ttime, \ldots , \state_\tHorizon)]
 = & \E^\policy_{\state_\ttime}
-\E^\policy_{\action_\ttime}[\cost(\state_\ttime,\action_\ttime)+\Cost_{\ttime+1}(f_\ttime(\state_\ttime,\action_\ttime))]\\
+\E^\policy_{\state_\ttime,\action_\ttime}[\cost_{\ttime}(\state_\ttime,\action_\ttime)+\Cost_{\ttime+1}(f_\ttime(\state_\ttime,\action_\ttime))]\\
```
Comment (avivt) [inline chapter2-ddp.tex:422]: I don't understand the first change.

**Review by avivt — CHANGES_REQUESTED** [review]

File: chapter2-ddp.tex (hunk 1 above)
Comment (ynahum) [reply chapter2-ddp.tex:308]: OK. will fix.  BTW, do we assume no cost on terminal state? as it's also not part of the sum.

File: chapter2-ddp.tex (hunk 2 above)
Comment (ynahum) [reply chapter2-ddp.tex:422]: From what I understand, the expectation is given s_t and a_t (smoothing theorem of expectations).
so it's more accurate to add both and not only s_t. The first expectation is still given s_t only.

**Comment by ynahum** [comment]
Hi Aviv
Kindly reminder :)

File: chapter2-ddp.tex (hunk 1 above)
Comment (avivt) [reply chapter2-ddp.tex]: yes, since the cost is time dependent, it does not matter.

File: chapter2-ddp.tex (hunk 2 above)
Comment (avivt) [reply chapter2-ddp.tex:421]: I think it's OK as it is. The first expectation conditions on the state, so in the second )inner) expectation the state is a constant, and we condition on the action.

File: chapter3-MDP-FH.tex [hunk 3]
```diff
@@ -441,19 +441,19 @@ \subsection{Stochastic Shortest-Path Problems}
...
 Stochastic shortest path includes also the discounted infinite
 horizon. To see that, add a new goal state, and from each state with
 probability $1-\discount$ jump to the goal state and terminate. The
 expected return of a policy would be the same in both models.
 Specifically, we add a state $\state_G$, such that
-$p(\state_G|\state,\action)=1-\discount$, for any state
-$\state\in\States$ and action $\action\in\Actions$ and
-$p(\state'|\state,\action)=\discount p(\state'|\state,\action)$. The
+$p(\state_G|\state,\action)=1-\discount$ for any state
+$\state\in\States/\{\state_G\}$ and action $\action\in\Actions$, while
```
Comment (avivt) [inline chapter3-MDP-FH.tex:452]: don't need that because S is defined without s_G

**Review by avivt — CHANGES_REQUESTED** [review]

File: chapter2-ddp.tex (hunk 2 above)
Comment (ynahum) [reply chapter2-ddp.tex]: reverted in a follow up commit

File: chapter3-MDP-FH.tex (hunk 3 above)
Comment (ynahum) [reply chapter3-MDP-FH.tex]: reverted in a follow up commit

**Comment by avivt** [comment]
Thanks a lot!

**Pull request merged**